src/requestVariableCompletionItemProvider.ts

```diff
@@ -67,6 +74,19 @@ export class RequestVariableCompletionItemProvider implements CompletionItemProv
         return completionItems;
     }
 
+    private checkIfRequestVariableDefined(document: TextDocument, variableName: string) {
+        const text = document.getText();
+        const regex = new RegExp(Constants.RequestVariableDefinitionRegex, 'mg');

```

Comment:
Why not directly construct a regex with given variableName, so that if we can found one in `text` means `True`, otherwise `False`.